### PR TITLE
BC-7282 - Adjust Sidebar Toggle Button Placement

### PR DESCRIPTION
**Pull request by MartinSchuhmacher**
# Short Description

For a smoother transition of the sidebar we want to adjust the cloud logo and toggle button placement.

## Links to Ticket and related Pull-Requests
BC-7282
https://github.com/hpi-schul-cloud/schulcloud-client/pull/3453

## Changes
- adjusting logo and expand menu dependend on sidebar status
- move colors to base options file
- fix opacity on user menu

## Screenshots of UI changes
OPEN sidebar
grafik

CLOSED sidebar
grafik

## Checklist before merging

- [x] QA: In addition to review, the code has been manually tested (if manual testing is possible)
- [x] PO: Any deviation from requirements was agreed with Product-Owner / ticket author / support-team
- [x] DEV: Every new component is implemented having accessibility in mind (e.g. aria-label, role property)

> Notice: Please keep this Pull-Request as a Draft (or add WIP label), until it is ready to be reviewed

### CODE REVIEW

File: src/modules/ui/layout/topbar/TopbarLogo.vue [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+</template>
+
+<script setup lang="ts">
+import { useI18n } from "vue-i18n";
+import { ENV_CONFIG_MODULE_KEY, injectStrict } from "@/utils/inject";
+
+const { t } = useI18n();
+const envConfigModule = injectStrict(ENV_CONFIG_MODULE_KEY);
+
+const altText = envConfigModule.getEnv.SC_TITLE + " " + t("global.topbar.logo");
```
Comment (odalys-dataport) [inline src/modules/ui/layout/topbar/TopbarLogo.vue:14]: Nice

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

File: src/modules/ui/layout/topbar/TopbarLogo.vue [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<template>
```
[the remark this answers is not in the document]
Comment (MartinSchuhmacher) [reply src/modules/ui/layout/topbar/TopbarLogo.vue:1]: you are right, will change that after waiting for maybe more feedback from Omar

**Review by EzzatOmar — APPROVED** [review]
I haven't found any issues.
Approved :)

File: src/locales/uk.ts [hunk 3]
```diff
@@ -760,6 +760,7 @@ export default {
 	"format.dateUTC": "JJJJ-MM-DD",
 	"format.dateYY": "DD.MM.YY",
 	"format.time": "HH:mm",
+	"global.cloudlogo": "Логотип",
```
Comment (odalys-dataport) [inline src/locales/uk.ts:763]: These should be written like "cloudLogo" :D

File: src/modules/ui/layout/topbar/Topbar.vue [hunk 4]
```diff
@@ -1,10 +1,12 @@
...
 	<VAppBar flat>
+		<CloudLogo v-if="!sidebarExpanded" class="mt-1" />
 		<template v-slot:prepend>
 			<VAppBarNavIcon
-				:icon="sidebarToggleIcon"
+				:icon="mdiMenu"
 				@click="() => $emit('sidebar-toggled')"
-				size="small"
+				size="default"
+				v-if="!sidebarExpanded"
```
Comment (odalys-dataport) [inline src/modules/ui/layout/topbar/Topbar.vue:9]: Please put the `v-if` at the beginning. This should be done by our formatter, no idea why it isn't doing that anywhere.

**Review by odalys-dataport — CHANGES_REQUESTED** [review]

**Review by odalys-dataport — APPROVED** [review]

**Pull request merged**